framework/src/version.ts

```diff
@@ -12,8 +12,4 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-'use strict';
-
-const Node = require('./node');
-
-module.exports = { Node };
+export const version = '0.1.0';
```

Comment:
is this version number correct? not `0.6.0`?

Reply:
I think this need to be re considered as it is never used, but this is correct for the typescript conversion

Reply:
@nazarhussain do we still need this? I am not sure when we will use this and upgrade it